packages/storage/src/cookies.ts

```diff
@@ -72,7 +73,7 @@ try {
  *   httpOnly?: boolean;
  *   maxAge?: number;
  *   sameSite?: "None" | "Lax" | "Strict";
- *   getRequest?: () => Request | () => PageEvent // (useRequest from solid-start)
+ *   getRequest?: () => Request | () => PageEvent // useRequest from solid-start, vite users must pass the "useRequest" from "solid-start/server" function manually
```

Comment:
Yes I had, because my vite doesn't support require, can you share your stacblitz so i can get a closer look at the differences?

Reply:
Please see https://github.com/solidjs-community/solid-primitives/issues/480

Reply:
Ok, but than this comment is 100% ligit, because it only works for older vite users. If it does work without setting `getRequest` nobody has been harmed, but if it doesn't there is a peice of documentation that informs you about that

Reply:
It's also NOT working for me in Solid-Start, locally or in StackBlitz.

Reply:
you mean without providing the `getRequest` manually?

Reply:
> you mean without providing the `getRequest` manually?

Yes.

Reply:
@atk please can you share your environment so that we can take a look, why it is working on your side